pages/place/routers/amend.js

```diff
@@ -13,15 +12,6 @@ module.exports = settings => form(Object.assign({
       .then(() => next())
       .catch(next);
   },
-  locals: (req, res, next) => {
-    res.locals.static.schema = req.form.schema;
-    getNacwoById(req, req.form.values.nacwo)
-      .then(nacwo => {
-        res.locals.model.nacwo = nacwo;
-      })
-      .then(() => next())
-      .catch(next);
-  },
```

Comment:
This seems weirdly out of scope. What's occurring?

Reply:
the fieldset was expecting an object for nacwo, I removed this so we always deal with the id except when rendering